op-defender/psp_executor/README.md

```diff
@@ -0,0 +1,24 @@
+### PSP Executor Service
+
+The PSP Executor service is a service designed to execute PSP onchain faster to increase our readiness and speed in case of incident response.
+
+The service is design to listen on a port and execute a PSP onchain when a request is received.
+
+⚠️ The service as to use a authentification method before calling this API ⚠️
+
+```
+OPTIONS:
+   --rpc-url value             Node URL of a peer (default: "http://127.0.0.1:8545") [$PSPEXECUTOR_MON_NODE_URL]
+   --privatekey value          Private key of the account that will issue the pause () [$PSPEXECUTOR_MON_PRIVATE_KEY]
```

Comment:
Probably is good to have not only the key, but also some option to mention where to take the key from. 
If not implemented yet this is fine for now.